# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mol/ppc64.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- trunk/src/molelf/Makefile.vanilla	2011-03-17 14:57:11.000000000 +0000
+++ trunk/src/molelf/Makefile	2011-03-17 14:58:07.000000000 +0000
@@ -18,6 +18,7 @@
 
 LINKCMD-$(LINUX)	= $(LD) -Ttext=0x101000 -Bstatic $^ -o $@
 LINKCMD-$(OSX)		= $(LD) -e __start -static $^ -o $@
+ASFLAGS			+= -mppc32
 CFLAGS			+= -static $(NOSTACKPROTECT)
 
 $(ODIR)/selftest: $(ODIR)/init.o $(ODIR)/libxselftest.a
